torchrl/envs/transforms/transforms.py

```diff
@@ -8557,3 +8557,157 @@ def _inv_call(self, tensordict):
         if self.sampling == self.SamplingStrategy.RANDOM:
             action = action + self.jitters * torch.rand_like(self.jitters)
         return tensordict.set(self.in_keys_inv[0], action)
+
+
+class AbsorbingStateTransform(ObservationTransform):
```

Comment:
We need tests for this class
It should be registered in the `__init__.py` and put in the doc.